src/python_testing/TC_CLCTRL_6_1.py

```diff
@@ -0,0 +1,820 @@
+            logging.info("Positioning feature supported or Instantaneous feature not supported, skipping steps 5b to 5f")
+
+            # Skipping steps 5b to 5f
+            self.skip_step("5b")
+            self.skip_step("5c")
+            self.skip_step("5d")
+            self.skip_step("5e")
+            self.skip_step("5f")
+        else:
+            logging.info("Positioning feature not supported and Instantaneous feature supported processing steps 5b to 5f.")
+
+            # STEP 5b: TH sends command MoveTo with Position = MoveToFullyOpen
+            self.step("5b")
+
+            sub_handler.reset()
+            try:
+                await self.send_single_cmd(cmd=Clusters.ClosureControl.Commands.MoveTo(
+                    position=Clusters.ClosureControl.Enums.TargetPositionEnum.kMoveToFullyOpen,
+                ), endpoint=endpoint)
+            except InteractionModelError as e:
+                asserts.assert_equal(
+                    e.status, Status.Success, f"Failed to send command MoveTo: {e.status}")
+                pass
+
+            # STEP 5c: Wait until TH receives a subscription report with OverallCurrentState.Position = FullyOpened.
+            self.step("5c")
+
+            logging.info("Waiting for OverallCurrentState.Position to be FullyOpened and OverallCurrentState.SecureState to be False")
+            sub_handler.await_all_expected_report_matches(expected_matchers=[current_position_matcher(Clusters.ClosureControl.Enums.CurrentPositionEnum.kFullyOpened)],
+                                                          timeout_sec=timeout)
+
+            # STEP 5d: TH sends command MoveTo with Position = MoveToFullyClosed
+            self.step("5d")
+
+            event_sub_handler.reset()
+            try:
+                await self.send_single_cmd(cmd=Clusters.ClosureControl.Commands.MoveTo(
+                    position=Clusters.ClosureControl.Enums.TargetPositionEnum.kMoveToFullyClosed,
+                ), endpoint=endpoint)
+            except InteractionModelError as e:
+                asserts.assert_equal(
+                    e.status, Status.Success, f"Failed to send command MoveTo: {e.status}")
+                pass
+
+            # STEP 5e: Verify that the DUT has emitted the MovementCompleted event.
+            self.step("5e")
+
+            # Wait for the MovementCompleted event to be emitted
+            event_sub_handler.wait_for_event_report(Clusters.ClosureControl.Events.MovementCompleted, timeout_sec=timeout)
+
+            # STEP 5f: TH reads from the DUT the MainState attribute
+            self.step("5f")
+
+            if attributes.MainState.attribute_id in attribute_list:
+                main_state = await self.read_clctrl_attribute_expect_success(endpoint, attributes.MainState)
+                logging.info(f"MainState: {main_state}")
+                asserts.assert_true(main_state == Clusters.ClosureControl.Enums.MainStateEnum.kStopped,
```

Comment:
When we are validating equality, we shoudl use assert_equals. Easy to understand the error during certification.